trieste/acquisition/function/entropy.py

```diff
@@ -41,7 +41,10 @@
 CLAMP_LB = 1e-8
 
 
-class MinValueEntropySearch(SingleModelAcquisitionBuilder[ProbabilisticModel]):
+T = TypeVar("T", bound=ProbabilisticModel, contravariant=True)
```

Comment:
In a separate PR I'd like to go over all the type variables in trieste and decide on more user friendly names (but still consistent and short)!